### ISSUE TITLE
Preserve comments from wit file in the binary

### ISSUE THREAD

**Issue by lu4p** [bytecodealliance/cargo-component#229]
Comments from the wit file are already, copied to the bindings.rs file.

These should also be contained in the final component, so that `wasm-tools component wit` can provide some useful documentation on the component.

**Comment by pchickey** [comment]
I agree this would be a valuable feature! This is an issue for the component model spec - it has been discussed https://github.com/WebAssembly/component-model/issues/213 but has not yet been implemented yet.
